getting_started.md

```diff
@@ -287,6 +287,43 @@ use the [analysis runner](https://github.com/populationgenomics/analysis-runner)
 which builds a Batch pipeline from a specific commit in a GitHub repository by
 running the driver itself on Hail Batch.
 
+### Analysis runner
+
+The analysis-runner builds a Batch pipeline from a speific commit in a GitHub repository. You can interact with an analysis server with the CLI. This can be installed with:
```

Comment:
```suggestion
The analysis-runner builds a Batch pipeline from a specific commit in a GitHub repository. You can interact with an analysis server with the CLI. This can be installed with:
```